views/js/app.js

```diff
@@ -16,48 +16,491 @@
+                $scope.alt_clock = data.expires;
+                $scope.alt_tick_interval = $interval($scope.alt_tick, 1000);
+                $scope.altsymbol = altsymbol;
+                $scope.altcoinselect = $scope.altcoins[altsymbol];
+                $scope.spinner = false;
+
+                process_alt_response(data);
+                //Fetch the order id using bitcoin address
+            });
+    }
+
+    //Create the altcoin order
+    function create_order(altcoin, amount, address, order_id) {
+        (function(promises) {
+            return new Promise((resolve, reject) => {
+                //Wait for both the altcoin limits and new altcoin order uuid
+                Promise.all(promises)
+                    .then(values => {
+                        $scope.order_id = order_id;
+                        //Hide the spinner
+                        $scope.spinner = false;
+                        var alt_minimum = values[0].min;
+                        var alt_maximum = values[0].max;
+                        //Compare the min/max limits for altcoin payments with the order amount
+                        if(amount <= alt_minimum) {
+                            //Order amount too low for altcoin payment
+                            update_altcoin_status('low_high');
+                            $scope.lowhigh = 'low';
+                            //Promise is run outside of the turn Angular sees so we need to tell
+                            //Angular to update all of our bindings as data has changed
+                            $scope.$apply();
+                        }else if(amount >= alt_maximum) {
+                            //Order amount too high for altcoin payment
+                            update_altcoin_status('low_high');
+                            $scope.lowhigh = 'high';
+                            //Promise is run outside of the turn Angular sees so we need to tell
+                            //Angular to update all of our bindings as data has changed
+                            $scope.$apply();
+                        }else{
+                            var uuid = values[1].order.uuid;
+                            //Save the altcoin uuid to database
+                            AltcoinAjax.get({
+                                'action': 'save_uuid',
+                                'address': address,
+                                'uuid': uuid
+                            });
+                            //Fetch the order id
+                            AltcoinAjax.get({
+                                'action': 'fetch_order_id',
+                                'address': address
+                            },function(order_id) {
+                                $scope.id_order = order_id.id;
+                            });
+                            $scope.altuuid = uuid;
+                            $scope.refundlink = $scope.alt_refund_url(uuid);
```

Comment:
Remove unused variable and function

Reply:
Fix refund link variable name. fdc3751b98471674738297d7175f341f668b7b8c